packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.scss

```diff
@@ -0,0 +1,10 @@
+ovh-manager-pci-storages-database-edit-connector-component {
+  button.oui-tabs-tab,
```

Comment:
**question:** Is there any issue with `<oui-tabs>` component?
As we tend to avoid custom styles, I'm wondering if we can address it directly on the component style instead of applying it only here.

Reply:
There is no issue with the `oui-tabs` component, but the way it looks does not exactly fits the needs for this page, because some labels are long, and look way better with `break-spaces`.
The difficulty here is that we have no control on the content of the oui-tabs, as it comes from the backend

Reply:
Maybe the component style should handle long labels (if its allowed).
In that way, it can benefit other places where such issue might appears.

Reply:
I'm not sure that we want the same behavior everywhere. As you can see on this screen, we have long labels appearing on two lines here.

![image](https://user-images.githubusercontent.com/24541481/157427403-f7e046bc-d97e-4597-9913-470b299fcb38.png)

Without the custom css, some labels overflow: 

![image](https://user-images.githubusercontent.com/24541481/157427653-5d70df93-9346-4623-ac1a-29da3ca35dbc.png)

But I don't think we want to apply this custom css on other places (like here):

![image](https://user-images.githubusercontent.com/24541481/157426930-a2af447c-ab71-4a67-99b6-bfb705792e91.png)